src/models/discreteBar.js

```diff
@@ -220,6 +222,7 @@ nv.models.discreteBar = function() {
         height:  {get: function(){return height;}, set: function(_){height=_;}},
         forceY:  {get: function(){return forceY;}, set: function(_){forceY=_;}},
         showValues: {get: function(){return showValues;}, set: function(_){showValues=_;}},
+        rx:      {get: function(){return rx;}, set: function(_){rx=_}},
```

Comment:
"rx" isn't very descriptive, I'd like to name it something more like "cornerRadius" so it's clear without reading the code what it does.